src/Umbraco.Web.UI.Client/src/packages/data-type/modals/data-type-picker-flow/data-type-picker-flow-modal.token.ts

```diff
@@ -1,9 +1,6 @@
 import { UmbModalToken } from '@umbraco-cms/backoffice/modal';
 
-export interface UmbDataTypePickerFlowModalData {
-	/** @deprecated This property will be removed in Umbraco 15. */
-	submitLabel?: string;
-}
+export type UmbDataTypePickerFlowModalData = object;
```

Comment:
same here, use `never`

Reply:
I'm unable to change this one to `never`, as it has a knock-on effect with the [`UmbModalRouteRegistrationController.onSetup()` call](https://github.com/umbraco/Umbraco-CMS/blob/v16/dev/src/Umbraco.Web.UI.Client/src/packages/data-type/components/data-type-flow-input/data-type-flow-input.element.ts#L54).
I started to look into updating the types, but it starts to go quite deep into the modal routes codebase.

Reply:
All good